projects/kubernetes/autoscaler/Makefile

```diff
@@ -16,6 +16,18 @@ FIX_LICENSES=$(REPO)/cluster-autoscaler/LICENSE
 HAS_RELEASE_BRANCHES=true
 EXCLUDE_FROM_STAGING_BUILDSPEC=true
 
+HAS_HELM_CHART=true
+
+HELM_GIT_TAG=$(shell cat ./$(RELEASE_BRANCH)/HELM_GIT_TAG)
+HELM_TAG=$(subst cluster-autoscaler-chart-,,$(HELM_GIT_TAG))
+HELM_DIRECTORY=$(RELEASE_BRANCH)
+HELM_SOURCE_OWNER=kubernetes
+HELM_SOURCE_REPOSITORY=autoscaler
+HELM_DIRECTORY=charts/cluster-autoscaler
```

Comment:
You override it here and this setting looks more correct to me, the location of the chart in the repository

Reply:
thanks. 🤦  I was wondering how I had both misunderstood the variable's function and still had a working build. This would explain it.